packages/react/src/components/AccountSettings/ChangePassword/__tests__/__snapshots__/ChangePassword.test.tsx.snap

```diff
@@ -42,7 +42,7 @@ exports[`ChangePassword renders as expected 1`] = `
             <button
               aria-checked="false"
               aria-label="Show password"
-              class="amplify-button amplify-field-group__control amplify-field__show-password"
```

Comment:
@Rogue-striker Looking at the snapshot, I am taking back my naming suggestion coz that will be a breaking change for now. For a patch, I think we are good to use the previous name. Could you change back to use `amplify-field__show-password`? Sorry about the back and forth here :(